### DISCUSSION TITLE
No "Wait-Late" option for relax mod

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17146] [Ideas] [answered]
This is something I really hope makes it into the game. That being no wait-late option on relax.

What is Wait-Late? Wait-Late is where relax will not tap until your cursor has reached the hitobject. Removing wait-late adds more skill to the relax mod as you MUST be on time and accurate to hit the notes. I think this addition would be a welcome one for the relax community.

**Comment by peppy** [comment, marked as answer]
Please search.

https://github.com/ppy/osu/discussions/13665

**Reply by TheFantasticLoki** [reply]
My suggestion would remove BOTH late and early hits since it would only tap EXACTLY on time independent of your cursor position on the map.
